### DISCUSSION TITLE
@types/mime has been deprecated, but still referenced by @types/serve-static

### DISCUSSION THREAD

**Discussion by huaxinjiayou** [DefinitelyTyped/DefinitelyTyped#69229] [Issues with a @types package] [answered]
please remove the @types/mime dependency in the project(like @types/serve-static), otherwise the compilation will report an error

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/41aa59f8e323a93b98ecea69fcf67a7a0a2cfa67/types/serve-static/package.json#L10C10-L10C23

<img width="442" alt="image" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/1059680/802cb2b4-847f-48b7-a032-0c9929ea8545">

<img width="1269" alt="image" src="https://github.com/DefinitelyTyped/DefinitelyTyped/assets/1059680/18ecb0b7-6760-4b5d-b6ff-c499adc26c6f">

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
Fixed in #69231
